provisioning/transport/mqtt/src/Microsoft.Azure.Devices.Provisioning.Transport.Mqtt.csproj

```diff
@@ -27,6 +51,9 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="DotNetty.Codecs.Mqtt" Version="0.4.7" />
```

Comment:
>7 [](start = 66, length = 1)

device client is using 0.4.6...we should align on all clients

Reply:
Sure, I'll try to increase the DeviceClient version as well and see if that passes the tests.

Reply:
This is more complicated than I originally thought. Tracking with #250.